### PR TITLE
fix: cleanup action release

### PR DESCRIPTION
**Pull request by snj33v**
## 1. Overview

* Upload artifacts to release
* remove the  release on clean command

### CODE REVIEW

**Review by ajeet97 — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,13 +27,11 @@ jobs:
       - name: Launch release process
         run: |
           make release-build-platform PLATFORM=amd64
-          make release-build-platform PLATFORM=arm64
```
Comment (puneet2019) [inline .github/workflows/release.yml:30]: why was this arm? haha

**Review by puneet2019 — APPROVED** [review]

**Review by Anmol1696 — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -27,13 +27,11 @@ jobs:
...
-        uses: actions/upload-artifact@v2
-        if: github.event.release
+      - name: attach binaries
+        uses: softprops/action-gh-release@v1
         with:
-          name: release
-          path: release
+          files: release/*
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Anmol1696) [inline .github/workflows/release.yml:37]: ```Error: ⚠️ GitHub Releases requires a tag```

Do we need this token? The GH action does not seem to be working

**Review by Anmol1696 — CHANGES_REQUESTED** [review]
not working yet

File: .github/workflows/release.yml (hunk 2 above)
Comment (snj33v) [reply .github/workflows/release.yml:37]: issue is a bit different, but yes action requires token for uploading files to release

Comment (snj33v) [reply .github/workflows/release.yml:38]: It is strange that this action was triggered, since we cannot always tag something in a PR.

**Review by ajeet97 — APPROVED** [review]

**Review by puneet2019 — APPROVED** [review]

**Pull request merged**